src/MainWindow.cpp

```diff
@@ -247,6 +249,24 @@ MainWindow::MainWindow(QWidget* parent):
 
     // start centered
     move(QApplication::desktop()->screen()->rect().center() - rect().center());
+
+    QAction* genBashRemediation = new QAction("&bash", this);
+    QObject::connect(
+        genBashRemediation, SIGNAL(triggered()),
+        this, SLOT(generateBashRemediationRole())
+    );
+    QAction* genAnsibleRemediation = new QAction("&ansible", this);
+    QObject::connect(
+        genAnsibleRemediation, SIGNAL(triggered()),
+        this, SLOT(generateAnsibleRemediationRole())
+    );
+
+    QMenu* remediationButtonMenu = new QMenu(this);
+    remediationButtonMenu->addAction(genBashRemediation);
+    remediationButtonMenu->addAction(genAnsibleRemediation);
+    // remediationButtonMenu->addAction(mGenPuppetRemediation);
+    // remediationButtonMenu->addAction(mGenAnacondaRemediation);
```

Comment:
the puppet command probably makes sense but anaconda roles don't have a use-case. I suggest to uncomment what's supposed to be there and remove anaconda action for good.